blockchain/source/migrations/9_set_allowance_to_market.js

```diff
@@ -0,0 +1,14 @@
+var SNMD = artifacts.require('./SNMD.sol');
+var Market = artifacts.require('./Market.sol');
+
+module.exports = function (deployer, network) {
+    if (network === 'private') {
+        SNMD.at('0x26524b1234e361eb4e3ddf7600d41271620fcb0a')
+            .then(function (token) {
+                token.AddMarket(Market.address, { gasPrice: 0 });
```

Comment:
Looks weird, a file is called `set_allowance_to_market` but this call doesn't look like a setting of allowance.